app.js

```diff
@@ -57,7 +57,7 @@ function processLine (line, prefix, defaultTags) {
     if (process.env.DEBUG) {
       console.log('Processing dyno metrics');
     }
-    let tags = tagsToArr({ dyno: line.source });
+    let tags = tagsToArr({ dyno: line.source, dynotype: line.source.split('.')[0] });
```

Comment:
line.source will never be null, right?

Reply:
I don't think so, but I don't know, and adding logic to guard against it is easy